### ISSUE TITLE
Static content returning 404 errors

### ISSUE THREAD

**Issue by alandtse** [spdx/spdx-online-tools#261]
Loading the new site at https://tools.spdx.org/ returns a broken page that appears to be missing resources form /static/. Tested in both chrome and firefox.

Here's my chrome console logs:
```text
19:21:23.673 tools.spdx.org/:15 GET https://tools.spdx.org/static/css/bootstrap.css net::ERR_ABORTED 404 (Not Found)
19:21:23.688 tools.spdx.org/:16 GET https://tools.spdx.org/static/css/starter-template.css net::ERR_ABORTED 404 (Not Found)
19:21:23.720 tools.spdx.org/:186 GET https://tools.spdx.org/static/js/jquery.min.js net::ERR_ABORTED 404 (Not Found)
19:21:23.722 tools.spdx.org/:187 GET https://tools.spdx.org/static/js/bootstrap.min.js net::ERR_ABORTED 404 (Not Found)
19:21:23.749 tools.spdx.org/:173 GET https://tools.spdx.org/static/images/logo_spdx.png 404 (Not Found)
```

Screenshot:
![image](https://user-images.githubusercontent.com/7086117/99754267-e4fd4380-2a9c-11eb-992b-ecdf583c72b7.png)

**Comment by goneall** [comment]
@alandtse Thanks for posting this - I was able to duplicate the problem after clearing my browser cache.

**Comment by goneall** [comment]
Resolved with PR #258 commit https://github.com/spdx/spdx-online-tools/pull/258/commits/f43b105b3841e1f057f8255615effd72ddb0c09d

Thanks again @alandtse for pointing this out so quickly!
